src/hmacKey.ts

```diff
@@ -193,62 +246,69 @@ export class HmacKey extends ServiceObject<HmacKeyMetadata | undefined> {
+       */
+      /**
+       * @callback HmacKeyMetadataCallback
        * @param {?Error} err Request error, if any.
-       * @param {HmacKey} hmacKey this {@link HmacKey} instance.
+       * @param {HmacKeyMetadata} metadata The updated {@link HmacKeyMetadata} object.
        * @param {object} apiResponse The full API response.
        */
       /**
-       * @typedef {array} GetHmacKeyResponse
-       * @property {HmacKey} 0 This {@link HmacKey} instance.
+       * @typedef {array} HmacKeyMetadataResponse
+       * @property {HmacKeyMetadata} 0 The updated {@link HmacKeyMetadata} object.
        * @property {object} 1 The full API response.
        */
       /**
-       * @typedef {object} GetHmacKeyOptions
-       * @property {string} [userProject] This parameter is currently ignored.
-       */
-      /**
-       * Retrieves and populate an HMAC key's metadata, and return
-       * this {@link HmacKey} instance.
-       *
-       * HmacKey.get() does not give the HMAC key secret, as
-       * it is only returned on creation.
+       * Updates the state of an HMAC key. See {@link SetHmacKeyMetadata} for
+       * valid states.
        *
-       * The authenticated user must have `storage.hmacKeys.get` permission
-       * for the project in which the key exists.
-       *
-       * @method HmacKey#get
-       * @param {GetHmacKeyOptions} [options] Configuration options.
-       * @param {GetHmacKeyCallback} [callback] Callback function.
-       * @returns {Promise<GetHmacKeyResponse>}
+       * @method HmacKey#setMetadata
+       * @param {SetHmacKeyMetadata} metadata The new metadata.
+       * @param {SetHmacKeyMetadataOptions} [options] Configuration options.
+       * @param {HmacKeyMetadataCallback} [callback] Callback function.
+       * @returns {Promise<HmacKeyMetadataResponse>}
        *
        * @example
        * const {Storage} = require('@google-cloud/storage');
        * const storage = new Storage();
        *
-       * //-
-       * // Get the HmacKey's Metadata.
-       * //-
+       * const metadata = {
+       *   state: 'INACTIVE',
+       * };
+       *
        * storage.hmacKey('ACCESS_ID')
-       *   .get((err, hmacKey) => {
+       *   .setMetadata(metadata, (err, hmacKeyMetadata) => {
        *     if (err) {
        *       // The request was an error.
        *       console.error(err);
        *       return;
        *     }
-       *     // do something with the returned HmacKey object.
+       *     console.log(hmacKeyMetadata);
        *   });
        *
        * //-
        * // If the callback is omitted, a promise is returned.
        * //-
        * storage.hmacKey('ACCESS_ID')
-       *   .get()
+       *   .setMetadata(metadata)
        *   .then((data) => {
-       *     const hmacKey = data[0];
+       *     const hmacKeyMetadata = data[0];
+       *     console.log(hmacKeyMetadata);
        *   });
        */
-      get: true,
+      setMetadata: {
+        reqOpts: {
+          method: 'PUT',
```

Comment:
❤️

Reply:
Do we need to update the minimum `@google-cloud/common` version in package.json here?

Reply:
Oh yes, good catch.